.gitpod.yml

```diff
@@ -4,6 +4,10 @@ tasks:
         echo 'export PATH="$HOME/bin:$PATH"' >> ~/.bashrc
         sudo apt-get install -q -y libunwind-dev
         sh -ci "$(curl -fsSL https://internetcomputer.org/install.sh)"
+        curl -sLO https://download.dfinity.systems/ic/69e1408347723dbaa7a6cd2faa9b65c42abbe861/openssl-static-binaries/x86_64-linux/pocket-ic.gz
```

Comment:
We install latest dfx in gitpod, whereas for dev containers we use a fixed version. Consider setting the version here too.

Reply:
done